### PR TITLE
Spaces visual fixes

### PR DESCRIPTION
**Pull request by robintown**
Cut off long room names and align checkboxes in the add existing rooms dialog, and fix the invite button margin.

Before:

Screenshot_2021-04-22 Element Matrix(1)
Screenshot_2021-04-22 Element Toaq(2)

After:

Screenshot_2021-04-22 Element Matrix
Screenshot_2021-04-22 Element Toaq(1)

### CODE REVIEW

File: res/css/views/dialogs/_AddExistingToSpaceDialog.scss [hunk 1]
```diff
@@ -148,12 +148,13 @@ limitations under the License.
                 font-size: $font-15px;
                 line-height: 30px;
                 flex-grow: 1;
+                overflow: hidden;
+                text-overflow: ellipsis;
```
Comment (t3chguy) [inline res/css/views/dialogs/_AddExistingToSpaceDialog.scss:152]:
```suggestion
                text-overflow: ellipsis;
                white-space: nowrap;
```

Comment (t3chguy) [reply res/css/views/dialogs/_AddExistingToSpaceDialog.scss:152]: fixes 
image

**Review by t3chguy — APPROVED** [review]
Otherwise this is great and much appreciated, thanks

**Comment by t3chguy** [comment]
Please avoid force pushing as it discards my review thus far.

**Comment by robintown** [comment]
Ah, will do. Netlify is failing? :confused:

**Comment by t3chguy** [comment]
You need to pull latest develop

**Comment by robintown** [comment]
Should be good to go

**Comment by t3chguy** [comment]
Awesome, thanks for this!

**Pull request merged**